paper/mitchell-geophysics.tex

```diff
@@ -0,0 +1,934 @@
+
+The data quality control methodology presented, is potentially applicable to a variety of field surveys. Here we test it in an underground environment with DC resistivity data. We find that a combination of a few poor electrodes and current leakage problems within the cables, that connect electrodes, have conspired to generate a highly noise-contaminated dataset. Despite the highly contaminated nature of the dataset, we are able to use the outlined data QC methodology to identify a subset of reliable data for inversion and obtain an interpretable inversion model.
+
+In this paper, we open with a description of the case history, which provided the impetus for this research, and then present the 4-stage data QC methodology in the context of the case history. The final results of the field data inversion and the results of several synthetic problems are shown to illustrate the utility of the methodology. We conclude with a discussion of the results and the performance of the data QC methodology.
+
+\section{Motivating Case Study}
+\label{CaseStudy}
+
+Engineers at the Mosaic K2 potash mine outside of Esterhazy, Saskatchewan have been combating water inflow problems for over a decade with limited success. In an effort to learn more about their problem a large non-conventional DC resistivity dataset was collected. Using this data, we aim to accurately image the source of the water inflow and map the conduits through which the water enters the mine.
+
+The K2 mine has workings approximately 1 km beneath the surface, in the upper layers of the Middle Devonian (393-382 mya) Prairie Evaporite. Although several aquifers in the lower Dawson Bay Formation overlie the Prairie Evaporite, a roughly 20 m thick layer of evaporites and a dolomitic shale layer, known as the Second Red Bed, provide an impermeable membrane which keeps water from the permeable aquifers out of the mine. When the integrity of the Second Red Bed and the overlaying evaporite are weakened by naturally occurring voids and inhomogeneities or mining induced fracturing related to subsidence and ``creeping,'' water inflow problems often result. \cite{Gendzwill1992} and \cite{Chouteau1997} present a more detailed discussion of the types of problems encountered in potash mining and review many of the associated geophysical studies. Figure \ref{fig:Intro_ProblemCartoon} shows an overview of the mine environment and provides context regarding the geological setting.
+
+\begin{figure} [!ht]
+    \begin{center}
+       \includegraphics{./Figures/Fig1.png}

```

Comment:
I updated this one here to be a png. The rest of the figures will need to be updated to the `.png` extension once you have created them from the pdfs